archunit/src/main/java/com/tngtech/archunit/library/freeze/StoreReadException.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2019 TNG Technology Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.tngtech.archunit.library.freeze;
+
+class StoreReadException extends RuntimeException {
```

Comment:
But `IOException` is one of my beloved checked exceptions :wink: So yes, I would use `IOException` if I really think a user should be forced to deal with it. But how will you deal with it within an ArchUnit test? My guess would be to wrap it into a `RuntimeException` because there is not much you can do. And then there is no value to force the user to deal with it...

Reply:
Ah, never mind! (Yesterday night, I somehow thought that it was a `RuntimeException`...)